Special abilities v2, mostly Flak (#4)
GameBrains/er-core: modules/special-rules/pages/brood-telepathy.adoc

---

> Apply these rules to detachments with the {synapse} ability:

Reviewer: I've just spotted this after first making a comment below. Shouldn't this say {brood-telepathy}, per the topic title? And what does that mean for the below (I haven't absorbed the related changes enough — no doubt I should look closer another time when I'm in less of a rush. But I'm pretty sure this bit is in error).

Author: Yes!
```suggestion
Apply these rules to detachments with the {brood-telepathy} ability:
```

> * If the detachment includes any units with the {synapse-creature} ability:
>  ** The detachment has the {dauntless} ability so automatically passes all Leadership tests.
> 
> * If the detachment loses all its {synapse-creature} it loses the {unbreakable} ability.

Reviewer: ```suggestion
* While the detachment includes any {synapse-creature} units it gains the {dauntless} ability -- so, you will automatically pass all Leadership tests that you are required to make for this detachment.

* If the detachment loses all its {synapse-creature} units then it also loses the {unbreakable} ability.
```
1. I've suggested new wording, but do we really need to say anything similar to the "so, you will automatically pass all Leadership tests that you are required to make for this detachment" part? I mean, you need to know the Dauntless ability anyway, and we have the 'Related information' link as an aid.
2. The detachment doesn't actually _have_ the Unbreakable ability to lose, except by virtue of the Synapse rules. Or are you especially concerned that people will forget to drop the Unbreakable at the critical moment? (If so, then perhaps an IMPORTANT notice might be warranted here, phrased in a way that makes that connection without implying that the detachment actually had Unbreakable as an inherent ability.)

Author: 1. Agreed. Dauntless (and Unbreakable for the equivalent text in Synapse) is simple enough to not need summarising.
2. The detachment will always start with the Dauntless ability because there's no way to create a detachment that includes no Synapse Creatures. But I think your 'while x condition' construction does the job.

So:

```suggestion
* While the detachment includes any {synapse-creature} units it gains the {dauntless} ability.
```

Author: Hmm, slight further change:

```suggestion
* While the detachment includes any {synapse-creature} units it has the {dauntless} ability.
```

'Has' instead of 'gains' should reinforce that it's a temporary ability rather than gaining the ability from that point on.

Reviewer: I'm slightly confused I thought your points 1 & 2 were replies to my points 1 & 2 and you'd mixed Dauntless with Unbreakable — but from other context, I think you made your own new points 1 and 2, and part of what you're saying is that we don't need the second bullet, about Unbreakable? If so, then this sounds OK to me, if I've unpicked that right :-D.

As for 'has' versus 'gains'. Hmmm. 'Gains' is qualified by 'while' anyway. 'Benefits from' could've been another option, but that seems a bit off. Actually, I think part of the trouble is that it's all a bit passive. How about this:
```suggestion
* While the detachment includes any {synapse-creature} units, apply the {dauntless} ability.
```
...or perhaps "While the detachment includes at least one {synapse-creature} unit, apply the {dauntless} ability."
